packages/katello/katello/katello.spec

```diff
@@ -180,6 +180,12 @@ Provides a federation of katello services
 # the files section is empty, but without it no RPM will be generated
 
 %changelog
+
+* Tue Aug 11 2020 Jeremy Lenz <jlenz@redhat.com> - 4.0.0-0.4.master 
```

Comment:
Almost there. Can you change (https://github.com/theforeman/foreman-packaging/pull/5644/files#diff-524a56c89a7dd0e9dbe2443042774384R8) to be a `2` and then update this to be `3.17.0-0.2.master`

Reply:
@ehelms Change what to a `2`?  (The link goes to this commit)

Reply:
Assuming you meant https://github.com/theforeman/foreman-packaging/blob/f0dc385aa362ad2bbe0e74c49b4495b3fb11267f/packages/katello/katello/katello.spec#L8